From 259874d731a5a051027efa79d1acdf5df4893063 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Thu, 2 May 2019 17:49:21 +0200
Subject: [PATCH 131/152] workspace: Return results of forwarded acceptDrop

Return the results of calls to acceptDrop that we forwarded to the
Workspace object.

This fixes a bug where app icons that were dragged and released above a
window clone would get animated back to their original position
(indicating that nothing happened) even though they opened correctly.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/121
---
 js/ui/workspace.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/workspace.js b/js/ui/workspace.js
index 7ffec352b..92f9622f2 100644
--- a/js/ui/workspace.js
+++ b/js/ui/workspace.js
@@ -421,7 +421,7 @@ var WindowClone = GObject.registerClass({
     }
 
     acceptDrop(source, actor, x, y, time) {
-        this._workspace.acceptDrop(source, actor, x, y, time);
+        return this._workspace.acceptDrop(source, actor, x, y, time);
     }
 
     _onDragCancelled(_draggable, _time) {
-- 
2.23.0

